tests/dataset/test_anndata.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import random
+import pandas as pd
+import scipy.sparse as sparse
+
+import pytest
+from scvi.dataset._biodataset import BioDataset
+from scvi.dataset._datasets import synthetic_iid
+from scvi.dataset import (
+    setup_anndata,
+    transfer_anndata_setup,
+    register_tensor_from_anndata,
+)
+from scvi.dataset._anndata import get_from_registry
+
+
+def test_transfer_anndata_setup():
```

Comment:
these are important because they will break when there is a major api change in scanpy! And we will be the first notified of it :)